Updated function in ObjCB.js

android-dep opened the pull request:
Adjusted the chatNotice function to be sendNotice, see the attached PDF regarding functions.
[Chaturbate.pdf](https://github.com/brandonxavier/CB-AppDevKit/files/100241/Chaturbate.pdf)

android-dep committed:
Update ObjCB.js

android-dep committed:
Some minor adjustments

Added the new value for sendNotice where you can msg categories of users
by their color.
Added a dummy grey user in chat to test with.

agtg1966 commented:
u have nothing i want

On Thu, Jan 21, 2016 at 8:37 PM, android-dep notifications@github.com
wrote:

> Adjusted the chatNotice function to be sendNotice, see the attached PDF
> regarding functions.
> Chaturbate.pdf
> 
> ## https://github.com/brandonxavier/CB-AppDevKit/files/100241/Chaturbate.pdf
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/brandonxavier/CB-AppDevKit/pull/28
> Commit Summary
> - Update ObjCB.js
> 
> File Changes
> - _M_ runtime/ObjCB.js
>   https://github.com/brandonxavier/CB-AppDevKit/pull/28/files#diff-0
>   (7)
> 
> Patch Links:
> - https://github.com/brandonxavier/CB-AppDevKit/pull/28.patch
> - https://github.com/brandonxavier/CB-AppDevKit/pull/28.diff
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/brandonxavier/CB-AppDevKit/pull/28.

android-dep committed:
Just a few issues fixed to restore functionality

Although it's a far cry from testing on the testbed online, still helps
to test misc. apps\bot scripts on the local drive and it takes only a
few minutes to narrow down the issue of a script failing. Just trying to
keep this project from dying, it's a helpful tool in troubleshooting.

malamel commented:
Yes

malamel commented on runtime/ObjCB.js:
```diff
@@ -1,7 +1,9 @@
  * Created with JetBrains WebStorm
  * User: brandonxavier (brandonxavier421@gmail.com)
- * Date: 6/20/13
+ * Original Date: 6/20/13
```
Original

malamel commented on Docs/onLeave.txt:
```diff
@@ -0,0 +1,22 @@
+Notice: Bye testuser!
+Notice: {u'user': u'testuser', u'in_fanclub': False,
+         u'has_tokens': False, u'is_mod': False,
+         u'gender': u'm', u'tipped_recently': True}
```
True

android-dep committed:
Add files via upload
